scripts/cli.py

```diff
@@ -302,17 +289,20 @@ def deploy_to(controller, cloud, model, channel, build, overlays, password):
         'metadata': {'name': 'ambassador', 'namespace': model},
         'spec': {'tls': [{'hosts': [pub_addr], 'secretName': 'ambassador-tls'}]},
     }
-    result = subprocess.run(
-        ['kubectl', 'apply', '-f', '-'],
-        input=yaml.dump(patch).encode('utf-8'),
-        stderr=subprocess.PIPE,
-        stdout=subprocess.PIPE,
-    )
-    try:
-        result.check_returncode()
-    except subprocess.CalledProcessError as err:
-        print("Couldn't set Ambassador up properly:")
-        print(err)
+
+    for _ in range(12):
```

Comment:
are you doing this to sleep a max of 1 minute?  could you, ya know, say that somewhere?